### DISCUSSION TITLE
為什麼分手後還會一直關注前任

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1544] [General]
和愛人見面時，多巴胺會為你帶來幸福感﻿，直到離別後，因為多巴胺不再繼續送幸福﻿，那份渴望遂化做思念，使你感到好想他喔﻿
﻿
可要是你在這時候不幸被甩﻿
這套多巴胺的啟動機制並不會說斷就斷﻿
依舊會在看到愛人時湧起戀愛時的愉悅感﻿，這就是為什麼有些人分手後還無法斷捨離﻿，甚至還要關心前任消息、偷看他的限時動態：﻿
﻿
因為這整個過程就像戒毒一樣﻿
你的大腦仍在懷念多巴胺釋放時的愉悅感﻿，你一直想著再來一口、再一口就好﻿

我覺得分手後就跟戒毒一樣應該要是循序漸進的，一開始還是可以關注對方動態，但可以慢慢減量，到最後才完全脫離。

**Comment by aa103967622** [comment]
以我的經驗會想起前任，多半是想起以前快樂的相處時光，而不是她本人，比如說，你們第一次牽手，或是第一次約會，每一個細節你都一定記憶得清清楚楚，這些都在你腦中揮之不去，但你想念的只會是那些與她相處的時光，而非她本人。
